pull-request/src/person.js

```diff
@@ -2,7 +2,7 @@ function Person(first, last) {
     this.firstName = first;
     this.lastName = last;
     this.age = null;
-    this.eyeColor = null;
+    this.eyeColor = undefined;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Use null instead. ([javascript:S2138](http://localhost:9750/coding_rules#rule_key=javascript%3AS2138))

[See it in SonarQube](http://localhost:9750/project/issues?id=training%3Apull-request&pullRequest=40&issues=AWgUWU8K6PH_c9BW2lvs&open=AWgUWU8K6PH_c9BW2lvs)

[ISSUE_KEY]: AWgUWU8K6PH_c9BW2lvs